application/src/main/kotlin/net/corda/v5/application/marshalling/MarshallingService.kt

```diff
@@ -4,31 +4,74 @@ package net.corda.v5.application.marshalling
 import net.corda.v5.base.annotations.DoNotImplement
 
 /**
- * Handles conversion of string data to and from strongly typed objects.
+ * [MarshallingService] marshalls to and from formatting string data.
  *
  * Corda provides a number of marshalling services for converting between string data in different formats. Users should
  * not ask for a [MarshallingService] directly but should instead use one of the specialized services that declare
  * what format to work with (e.g. [JsonMarshallingService] for working with JSON data).
+ *
+ * Example usage:
```

Comment:
I've provided the basically the same flow example in `RPCStartableFlow`. I don't think we need both and seeing it used in `RPCStartableFlow` is probably more useful. Should we remove this example?